src/DragHandle.tsx

```diff
@@ -19,6 +22,106 @@ const cellDragHandle = css`
 
 const cellDragHandleClassname = `rdg-cell-drag-handle ${cellDragHandle}`;
 
-export default function DragHandle(props: React.HTMLAttributes<HTMLDivElement>) {
-  return <div className={cellDragHandleClassname} {...props} />;
+interface Props<R, SR> extends Pick<DataGridProps<R, SR>, 'rows' | 'onRowsChange'> {
+  columns: readonly CalculatedColumn<R, SR>[];
+  selectedPosition: SelectCellState | EditCellState<R>;
```

Comment:
I don't think it's possible
```suggestion
  selectedPosition: SelectCellState;
```